libraries/AP_GPS/AP_GPS_SIRF.cpp

```diff
@@ -204,48 +204,48 @@ AP_GPS_SIRF::_accumulate(uint8_t val)
 /*
   detect a SIRF GPS
  */
-bool
-AP_GPS_SIRF::_detect(struct SIRF_detect_state &state, uint8_t data)
+bool AP_GPS_SIRF::_detect(struct SIRF_detect_state &state, uint8_t data)
```

Comment:
They were done on a separate commit, which is the right policy for these fixes :)